CONTRIBUTING.rst

```diff
@@ -31,10 +31,50 @@ in mind:
 How to contribute
 =================
 
-We use GitHub pull requests (PRs) to make improvements to the repository.
+We use GitHub pull requests (PRs) to make improvements to the repository. Here
+is a concise description of our recommended workflow:
```

Comment:
We may want to say: Here is a concise step-by-step description of our recommended workflow. It is worth noting that the workflow is not unique. In general, we suggest contributors creating a new branch in the downstream and work on it to merge to our `master` branch.